src/main/java/core/basesyntax/service/CsvFileReaderImpl.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.service;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+public class CsvFileReaderImpl implements FileReader {
+
+    @Override
+    public List<String> readFromFile(String path) {
+        List<String> transactions = new ArrayList<>();
+        Scanner scanner;
```

Comment:
Use **BufferedReader** and try-with resources
Or even better **nio** and **Files** class

Same for your **FileWriter**